src/components/Main/Modal/SettingModal/ThemeTab/ThemeTab.vue

```diff
@@ -74,10 +86,13 @@ export default defineComponent({
     // TODO: 色のバリデーション
     // TODO: カラーピッカー
 
-    return { state }
+    return {
+      state
+    }
```

Comment:
意味のないdiffはなくしておきたいです 🙇 
```suggestion
    return { state }
```